qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -312,13 +312,13 @@ public void allowsOnlyProperlyNamedLocalVariables() throws Exception {
                         "Name 'prolongations' must match pattern",
                         "Name 'very_long_variable_id' must match pattern",
                         "Name 'camelCase' must match pattern",
-                        "Name 'it' must match pattern '^id$|^[a-z]{3,12}$'.",
+                        "Name 'it' must match pattern",
                         "Name 'number1' must match pattern",
-                        "Name 'ex' must match pattern '^id$|^[a-z]{3,12}$'.",
-                        "Name 'a' must match pattern '^id$|^[a-z]{3,12}$'.",
-                        "Name 'ae' must match pattern '^ex|[a-z]{3,12}$'.",
-                        "Name 'e' must match pattern '^ex|[a-z]{3,12}$'.",
-                        "Name 'it' must match pattern '^id$|^[a-z]{3,}$'."
+                        "Name 'ex' must match pattern",
```

Comment:
@pinaf No. Users see output from checkstyle.  Note that this is a test case of Qulice.  This Java file is checking the output from checkstyle. We are testing Qulice here.